frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/affinity.tsx

```diff
@@ -183,8 +186,8 @@ export const NodeAffinity: React.FC<NodeAffinityProps> = ({ affinity, onChange,
 
   return (
     <dl>
-      <Tooltip content={REQUIRED_TOOLTIP}>
-        <dt>Required During Scheduling Ignored During Execution</dt>
+      <Tooltip content={t(`olm~${REQUIRED_TOOLTIP}`)}>
```

Comment:
I don't think this will work with i18next parser since we don't have a comment for it? It would be better to use

```ts
const requiredTooltip = t('olm~Required rules must be met before a pod can be scheduled on a node.');
```

inside the component.